src/ResultWriter/faultoutput.f90

```diff
@@ -550,12 +551,29 @@ SUBROUTINE calc_FaultOutput( DynRup_output, DISC, EQN, MESH, MaterialVal, BND, t
               DynRup_output%OutVal(iOutPoints,1,OutVars) = LocSV !OutVars =8
           ENDIF
           IF (DynRup_output%OutputMask(5).EQ.1) THEN
+
+              !add transient nucleation stress to fault output
+              IF (EQN%FL.EQ.103) THEN
+                 NucleationStressXYZ = MATMUL(T(1:6,1:6), EQN%NucleationStressInFaultCS(iBndGP,:,iFace))
+                 Tnuc = DISC%DynRup%t_0
+                 IF (time.LE.0) THEN
+                    Gnuc = 0d0
+                 ELSEIF (time.LT.Tnuc) THEN
+                    Gnuc = exp((time-Tnuc)**2/(time*(time-2*Tnuc)))
+                 ELSE
+                    Gnuc = 1d0
+                 ENDIF
```

Comment:
Why don't you use the Calc_Smoothstep function here?

Reply:
because it is called Calc_SmoothStepIncrement, that is it compute the increment of nucleation stress and not the nucleation stress.

Reply:
https://github.com/SeisSol/SeisSol/blob/master/src/Physics/NucleationFunctions.f90#L86

Reply:
ups